bindata/metallb.yaml

```diff
@@ -14,8 +95,8 @@ spec:
   template:
     metadata:
       annotations:
-        prometheus.io/port: "7472"
-        prometheus.io/scrape: "true"
+        prometheus.io/port: '7472'
```

Comment:
Not related to the PR, also below

Reply:
I believe this is for the commit from #8

Reply:
Right, my wrong